src/components/PostPageDetail/index.js

```diff
@@ -687,7 +688,7 @@ export default withInteractionsManaged(React.memo(PostPageDetailIdComponent));
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: '#fff',
+    backgroundColor: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing hardcoded color values with centralized color constants improves maintainability and consistency of the codebase.
```diff
- backgroundColor: '#fff',
+ backgroundColor: COLORS.white,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->